storybookjs/storybook
# feat: add Vue support to @addons/background

Issue:
fixes #2845
## What I did
added a new decorator export for `@addons/background/vue`

## How to test
`yarn test --core`

Is this testable with Jest or Chromatic screenshots? y
Does this need a new example in the kitchen sink apps? y
Does this need an update to the documentation? y

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
